pom.xml

```diff
@@ -2439,7 +2439,7 @@
       <properties>
         <alt-hadoop>mapr</alt-hadoop>
         <rat.excludeSubprojects>true</rat.excludeSubprojects>
-        <hive.version>2.1.1-mapr-1710</hive.version>
+        <hive.version>2.3.3-mapr</hive.version>
         <hbase.version>1.1.1-mapr-1602-m7-5.2.0</hbase.version>
         <hadoop.version>2.7.0-mapr-1707</hadoop.version>
```

Comment:
Update Hadoop version to 2.7.0-mapr-1808?

Reply:
Done.